app/views/welcome/categories.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="row">
+  <div class="col-sm-4">
+    <div class="content-container">
+      <%= image_tag('categories/original.jpg', :alt => 'original art category photo', :class => 'categoryImage') %>
```

Comment:
for accessibility it's better to describe what the photo is. If this is something that will be changed by your client, make sure to include it in the CMS